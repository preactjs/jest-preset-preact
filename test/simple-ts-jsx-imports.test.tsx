import { h, Fragment, FunctionalComponent } from 'preact';

describe('serializer', () => {
	it('should serialize simple component', () => {
		const App: FunctionalComponent = () => (
			<>
				<div>foo</div>
				<Fragment>
					<div>foo2</div>
				</Fragment>
			</>
		);
		expect(<App />).toMatchSnapshot();
	});
});
