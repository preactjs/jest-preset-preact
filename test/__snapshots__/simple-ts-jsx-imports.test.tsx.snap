// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`serializer should serialize simple component 1`] = `
  <div>foo</div>
  <div>foo2</div>
`;
